### ISSUE TITLE
WebJar request

### ISSUE THREAD

**Issue by mateullas** [webjars/webjars#1815]
We have developed a modification of dropzone library in this repo:
https://github.com/asicudl/dropzone_udl
In this repo there is the pom.xml according to WebJars contributing guidelines
It also contains the zip file with the corresponding files for the WebJar

**Comment by jamesward** [comment]
So you want to distribute a fork of the dropzone library via WebJars?  Where is that fork?

**Comment by mateullas** [comment]
Hi, James.

Surely I misunderstood the instructions for contributing a WebJar.
We have a web application that uses this library and we have added the
possibility to edit filename (a feature request from our users)
I have created a repo in GitHub with our code and I have included there
the pom.xml for our project following the guidelines in
https://www.webjars.org/contributing
If I understand well we should make a fork of
https://github.com/webjars/dropzone
and add there our pom.xml
Is that correct?

Thank you in advance and sorry for the annoyance,

Mateu

**Comment by jamesward** [comment]
Yeah, we don't usually package the contents directly into the repo, instead we pull the contents from an existing library, ideally one that has tagged, released artifacts.  If your change is one that should just be in dropzone, then I'd recommend sending them a Pull Request.  Once merged and released we can just bump the existing WebJar to the new version.  If the change isn't a good candidate for upstream merging, then we can fork the existing WebJar into a new one that points to your fork.

**Comment by mateullas** [comment]
Hi, James. 
We are going to follow your advice. We are testing our local dropzone modification. 
When our code pass all tests we will make a Pull Request to incorporate our changes to dropzone library.

**Comment by mateullas** [comment]
Hi, James.

As you suggested we have contacted dropzone project (currently in GitLab:
gitlab.com/meno/dropzone).
We have followed their instructions for developers and introduced an issue
to communicate that we have a feature request (the posssibility of
changing file names when uploading). In this issue we have described in
detail our modification.

We have introduced this issue a month ago and we have no response.
We are afraid that this project is no longer maintained currently,
as newer issues also do not have response.

What do you suggest we can do?

Thank you in advance,

Mateu

**Comment by jamesward** [comment]
We can deploy a fork of the dropzone library.  Can you tag a release in your fork and let me know where the fork is?
